### PR TITLE
S16E12 and S17E01

### PR DESCRIPTION
**Pull request by rprouse**
Added the next two events and updated the front page to include myself as an organizer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,51 +9,53 @@
...
-                
+
             </div>
             <div class="col-md-6">
-                <h2><i class="fa fa-calendar-check-o"></i> Upcoming Events</h2>                
-                
-                {% for post in site.posts limit:1 %}
+                <h2><i class="fa fa-calendar-check-o"></i> Upcoming Events</h2>
+
+                {% for post in site.posts limit:2 %}
```
Comment (rprouse) [inline index.html]: We will need to revert this to 1 after the December event.

Comment (rprouse) [inline index.html]: Oops, where?

Comment (rprouse) [inline index.html]: Seen, fixing...

**Comment by rprouse** [comment]
Name fixed, sorry ☹️

**Comment by bpoetz** [comment]
thanks Rob!

**Pull request merged**